packages/shared/components/popups/StakingConfirmation.svelte

```diff
@@ -30,7 +30,7 @@
 
     const activeAirdrops = $participationOverview.find((apo) => apo.accountIndex === accountToStake.index)?.participations.map((p) => getAirdropFromEventId(p.eventId)) || []
 
-    let airdropSelections: { [key in StakingAirdrop]: boolean } = {
+    const airdropSelections: { [key in StakingAirdrop]: boolean } = {
```

Comment:
This might need to be `let` since some of the values change, haven't tested this for certain though.

Reply:
You can change the contents of an object declared using const